tests/integration_tests/user_group_tests/test_group_preferences.py

```diff
@@ -31,7 +31,7 @@ def test_preferences_in_group(api_client: TestClient, unique_user: TestUser) ->
 
 
 def test_update_preferences(api_client: TestClient, unique_user: TestUser) -> None:
-    new_data = UpdateGroupPreferences(recipe_public=False, recipe_show_nutrition=True)
```

Comment:
Is there a reason for this change? `recipe_public` is `True` by default, so shouldn't we be testing that setting it to `False` works? This is a different field than the one you changed (`private_group`)

Reply:
Indeed, it is not related. I thought I saw a test on group structure but no. I will correct the test